tests/ingestion/test_hash.py

```diff
@@ -0,0 +1,96 @@
+import io
+import hashlib
+import pickle
+from typing import Any
+import polars as pl
+
+
+def hash_gtfs_rt_row_marked_for_removal_wip(row: Any) -> bytes:
+    """hash row from polars dataframe"""
+    return hashlib.md5(pickle.dumps(row), usedforsecurity=False).digest()
+
+
+def test_hash_column_return_datatype() -> None:
+    """
+    return type of hash_gtfs_rt_row of Tuple[bytes] was coerced to binary before, but polars changes in 1.35 -> 1.36
+    resulted in the type now returning List[Binary]. This test is to catch changes in underlying inferred types
+    for future library upgrades.
+
+    We are no longer coercing the type, so this might never catch.
+
+    Remove this test if we ever remove the hash_gtfs_rt_row type logic from ingestion.
+
+    Investigate using just the base polars hash instead...why no
+
+    Check that all are hashable. check that same columns hash to the same value
```

Comment:
Can you clean this up? The current docstring isn't clear about what's being tested
```suggestion
    It hashes the same columns hash to the same value.
    
    return type of hash_gtfs_rt_row of Tuple[bytes] was coerced to binary before, but polars changes in 1.35 -> 1.36
    resulted in the type now returning List[Binary]. This test is to catch changes in underlying inferred types
    for future library upgrades.
```